content/docs/guides/osm_resource_limits.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "OSM Resource Requests and Limits"
+description: "Resource requests and limits for OSM pods and deployments"
+type: docs
+weight: 2
+---
+
+<h3>Default Resource Requests and Limits Table</h3>
```

Comment:
Why do the defaults need to be documented separately given that we autogenerate install time defaults https://github.com/openservicemesh/osm/blob/main/charts/osm/README.md? Can these be copy-pasted into the docs website instead?